### DISCUSSION TITLE
dropbox

### DISCUSSION THREAD

**Discussion by davidsi** [blinksh/blink#1972] [Ideas]
Is it possible to add a feature so I can connect to my dropbox account. Much of my code repository is there not GitHub

**Comment by carloscabanero** [comment]
We have not gotten any requests to add Dropbox to the app. Blink is also designed to be used with remote machines, and not for local coding. That being said:

One way you could workaround using your repository is by linking with the Dropbox app. Blink has a command `link-files`, that would basically link the Blink filesystem to the Dropbox application one. That way then you can even use the `code` command to make changes to your repository.

But it is less than ideal and I cannot recommend it. The way to go is most of the time to have a real repository somewhere, even connecting to your laptop, etc... And Blink can also provide you with an instant dev environment through Blink Build, that makes things easier too.

**Comment by davidsi** [comment]
Thx for your reply. I understand it’s preferable to connect to another machine. And most of the time that is what I plan on doing. However when that is not practical, as I have the files offline from the Dropbox app it would be beneficial to access them the way files in the Files app can be. I’m not sure why in this case using the link-files command is not safeThxDavidOn Feb 23, 2024, at 11:22, carloscabanero ***@***.***> wrote:﻿
We have not gotten any requests to add Dropbox to the app. Blink is also designed to be used with remote machines, and not for local coding. That being said:
One way you could workaround using your repository is by linking with the Dropbox app. Blink has a command link-files, that would basically link the Blink filesystem to the Dropbox application one. That way then you can even use the code command to make changes to your repository.
But it is less than ideal and I cannot recommend it. The way to go is most of the time to have a real repository somewhere, even connecting to your laptop, etc... And Blink can also provide you with an instant dev environment through Blink Build, that makes things easier too.

**Reply by carloscabanero** [reply]
I didn't say it is not safe, or pardon the confusion. I said it is _not ideal and I would not recommend it_, just because it is more elaborate. But you can definitely use it and there are no security issues. We have users doing "link-files" all the time for their needs and it works just fine. I just prefer to have everything remote.

**Comment by davidsi** [comment]
Awesome. Thank youDavidOn Feb 23, 2024, at 14:19, carloscabanero ***@***.***> wrote:﻿
I didn't say it is not safe, or pardon the confusion. I said it is not ideal and I would not recommend it, just because it is more elaborate. But you can definitely use it and there are no security issues. We have users doing "link-files" all the time for their needs and it works just fine. I just prefer to have everything remote.
